asset/css/markbind.css

```diff
@@ -5,3 +5,13 @@ blockquote {
 .radio-list-item label {
     font-weight: inherit;
 }
+
+@media (max-width: 767px) {
+    .navbar-nav .open .dropdown-menu {
+        background-color: #fff;
+        border-bottom: 1px solid rgba(0, 0, 0, 0.15);
+    }
+    .navbar-inverse .navbar-nav .open .dropdown-menu > li > a {
+        color: #000;
+    }
```

Comment:
Can you reorder the 2 blocks in alphabetical order? i.e. `navbar-i...` before `navbar-n...`

Taking cue from [teammatesCommon.css](https://github.com/TEAMMATES/teammates/blob/60a9efd3b5f1c7e1e0d035160bf9ed1f53e5d800/src/main/webapp/stylesheets/teammatesCommon.css#L408), let's also maintain the newline between blocks.